lib/pbench/server/api/resources/users_api.py

```diff
@@ -471,17 +471,19 @@ def delete(self, username):
 
         try:
             User.delete(username)
-            self.logger.info("User entry deleted for user with username {}", username)
+            self.logger.info(
+                "User entry deleted for user with username: {}, by user: {}",
+                username,
+                user.username,
+            )
         except NoResultFound:
             self.logger.info(
                 "Delete operation on non-existent user '{}'", username,
             )
             abort(404, message="User not found")
-        except AdminUserDeletion:
-            self.logger.info(
-                "Delete operation on an admin user '{}'", username,
-            )
-            abort(403, message=f"User {username} can not be deletd")
+        except AdminUserDeletion as exc:
+            self.logger.info(str(exc(username)))
```

Comment:
What does it mean to do `exc(username)`?  Is the exception an already instantiated object?  How can we invoke its constructor again?

Reply:
Ah, fixed it